app/services/induction/change_induction_record.rb

```diff
@@ -12,7 +12,7 @@ def call
           start_date: time_now,
           school_transfer: false,
         }
-        new_record = induction_record.dup
+        new_record = InductionRecord.new(induction_record.attributes.except("id", "end_date"))
```

Comment:
I was wondering how amending induction records for transferred participants will look now, will we risk setting their induction records to no end date although they already transferred and are active with another provider? (If we withdrew them for instance from the old provider)